index.html

```diff
@@ -333,12 +333,240 @@
         };
       </pre>
       <p>
-        <b>Draft summary:</b><br>
-        The <code><a>RemotePlaybackAvailability</a></code> interface exposes
-        whether there is a device on the user's network able to play one of the
-        sources available for the associated media element. The <code>change</code>
-        event is fired each time when the <code>value</code> changes.
+        A <a><code>RemotePlaybackAvailability</code></a> object is associated with available <a>remote playback devices</a>
+        and represents the <dfn>remote playback device availability</dfn> for the media element. If the user
+        agent can <a>monitor the list of available remote playback devices</a> in the background (without
+        a pending request to <code><a for="RemotePlayback">connect()</a></code>), the 
+        <a><code>RemotePlaybackAvailability</a></code> object MUST be implemented by the user agent.
```

Comment:
There're two places where the spec says the promise is rejected then. Though it may be worth repeating for self-containment. Done.